javascript/javascript1/week1/smart-ease.js

```diff
@@ -0,0 +1,55 @@
+//Excercise 1: ageify
+function ageify(yearOfBirth, yearFuture) {
+    let age = yearFuture - yearOfBirth;
+    return "You will be " + age + " years old in " + yearFuture;
+}
+
+ageify(1992, 2099);
```

Comment:
instead of    _return "You will be " + age + " years old in " + yearFuture;_
use console.log("You will be " + age + " years old in " + yearFuture);